samples/19.customization-plain-ui/src/PlainWebChat.js

```diff
@@ -0,0 +1,86 @@
+import { connectToWebChat } from 'botframework-webchat-component';
+import React, { useState } from 'react';
+
+import Attachment from './Attachment';
+import SuggestedActions from './SuggestedActions';
+
+import getValueOrUndefined from './util/getValueOrUndefined';
+
+const PlainWebChat = ({
+  activities,
+  sendMessage
+}) => {
+  const [sendBoxValue, setSendBoxValue] = useState('');
+
+  return (
+    <div>
+      <ul>
+        { activities
+            // Currently, we only display activity of type "message"
+            .filter(({ type }) => type === 'message')
+            // We need to hide "postBack" message sent by the user
+            .filter(({ channelData: { postBack } = {}, from: { role } }) => !(role === 'user' && postBack))
+            // We normalize the activity:
+            // - Every activity should have an "attachments" array, consists of zero or more attachments;
+            // - If this is a "messageBack" message, we should use the "displayText",
+            //   because "text" is being submitted to bot, and "displayText" is what we use to override the bot displayed to the user.
```

Comment:
```suggestion
            //   because "text" is being submitted to bot, and "displayText" is what we use to override what the bot displays
            
             
              
               
                
                 
                  
                    to the user.
```